ioeffect-cats/src/main/scala/scalaz/ioeffect/catscompat/package.scala

```diff
@@ -0,0 +1,64 @@
+package scalaz.ioeffect
+
+import cats.effect
+import cats.effect.{ ConcurrentEffect, Effect }
+import cats.syntax.all._
+
+import scala.util.control.NonFatal
+
+package object catscompat extends RTS {
```

Comment:
I think I'd rather this was a regular object rather than a package object. Package objects have very strange and arcane implicit handling rules that impact compiler performance.

Reply:
also, you totally missed an opportunity to call this `catz`